packages/opentelemetry-instrumentation-openai/tests/test_completions.py

```diff
@@ -20,6 +20,25 @@ def test_completion(exporter, openai_client):
     assert open_ai_span.attributes.get("llm.completions.0.content")
 
 
+@pytest.mark.vcr
+async def test_async_completion(exporter, async_openai_client):
+    await async_openai_client.completions.create(
+        model="davinci-002",
+        prompt="Tell me a joke about opentelemetry",
+    )
+
+    spans = exporter.get_finished_spans()
+    assert [span.name for span in spans] == [
+        "openai.completion",
+    ]
+    open_ai_span = spans[0]
+    assert (
+        open_ai_span.attributes["llm.prompts.0.user"]
+        == "Tell me a joke about opentelemetry"
+    )
+    assert open_ai_span.attributes.get("llm.completions.0.content")
```

Comment:
I'm happy to split this into it's own PR, figured some test coverage for non-streaming async calls would also be beneficial.